w3sper.js/tests/contract_test.js

```diff
@@ -173,13 +174,62 @@ test("contract.tx/send + events.once: transfer.deposit", async () => {
 
     // Get decoded event (driver & bytes hidden by contract facade)
     const decoded = await deposit$;
-    console.log('[decoded event] "deposit" ->', decoded);
-
-    const amt = decoded?.value ?? decoded?.amount;
-    if (amt !== undefined) {
-      assert.equal(BigInt(amt), AMOUNT);
-    }
+    assert.equal(BigInt(decoded.value), AMOUNT);
```

Comment:
Now that I notice it: do we have an issue about avoid returning such values as `Number`s?